Dockerfile

```diff
@@ -36,9 +48,9 @@ RUN python manage.py collectstatic --noinput
 
 RUN touch `echo $RANDOM`.txt
 
-RUN python manage.py compress --extension=.haml --force
+#RUN python manage.py compress --extension=.haml --force
```

Comment:
I think that we need this one line to compress haml files

Reply:
I believe this failed unless Redis was present.  I will move this to the entrypoint.sh script and push these changes.